hw02-generics/src/main/java/homework/CustomerReverseOrder.java

```diff
@@ -0,0 +1,16 @@
+package homework;
+
+import java.util.LinkedList;
+
+public class CustomerReverseOrder {
+
+    private final LinkedList<Customer> customerLinkedList = new LinkedList<>();
```

Comment:
```suggestion
    private final Deque<Customer> customerLinkedList = new LinkedList<>();
```

Тип лучше объявлять минимально необходимым интерфейсом.

Reply:
поправаил